### PR TITLE
Small fulton fixes

### PR DESCRIPTION
**Pull request by Profakos**
I have been messing around with Fultons on a test server, and noticed a few unreported issues.

- Fixes lack of feedback on connecting to the beacon system. This only really was a problem when only one beacon existed in the world, but feedbacks are always nice. 
- Fixes runtimes caused by using the fulton system with no backpacks. This was caused by an OR in place of an AND. I have placed an extra null check just to be sure.
- Fixed the extraction point's description containing an outdated pack linking method
- Changed the New to an Initialize

What I couldn't fix is the check on line 57: sure, it prevents you from extracting something from your other hand, but you would still be able to extract something from your backpack, or from a box on the ground, which messes up the inventory icons, leaving behind phantom images of things. Perhaps change the check, to see if the target's location is directly on a turf?

### CODE REVIEW

**Comment by Profakos** [comment]
I am still unsure about the aspects of the new Initialization, I only need the mapload parameter in the proc if I want the Initialization to only run when the map has fully loaded, correct?

File: code/modules/mining/fulton.dm [hunk 1]
```diff
@@ -60,7 +61,7 @@ var/list/total_extraction_beacons = list()
 		to_chat(user, "<span class='notice'>You start attaching the pack to [A]...</span>")
 		if(do_after(user,50,target=A))
 			to_chat(user, "<span class='notice'>You attach the pack to [A] and activate it.</span>")
-			if(loc == user || istype(user.back, /obj/item/weapon/storage/backpack))
+			if(loc == user && user.back && istype(user.back, /obj/item/weapon/storage/backpack))
```
Comment (RemieRichards) [inline code/modules/mining/fulton.dm]: ```&& user.back``` isn't required, ```istype()``` returns false if its arguments are null, so it's fine.

Comment (Profakos) [inline code/modules/mining/fulton.dm]: Ah, thanks! I guess the || was the only error on that line.

**Comment by Iamgoofball** [comment]
he's coming too?

**Comment by lzimann** [comment]
The mapload argument is only needed if you have other args in the proc.

File: code/modules/mining/fulton.dm [hunk 2]
```diff
@@ -35,6 +35,7 @@ var/list/total_extraction_beacons = list()
 		if(!A)
```
Comment (lzimann) [inline code/modules/mining/fulton.dm]: While you are at this file, can you also change the input two lines above to be `input(..) as null|anything in possible_beacons` so it has a cancel button, please?

Comment (Profakos) [inline code/modules/mining/fulton.dm]: Done. This has the side effect though, the list pops up even when there is just only one beacon in the word.

Comment (Militaires) [inline code/modules/mining/fulton.dm]: if(!(possible_beacons.len = 1))

Comment (lzimann) [inline code/modules/mining/fulton.dm]: you mean `possible_beacons.len != 1`, but I don't really think always showing the input is a problem

**Pull request merged**